src/aibrary/resources/aibrary_sync.js

```diff
@@ -0,0 +1,48 @@
+const openai = require('openai');
+const axios = require('axios');
+const { AibraryChatCompletion } = require('./AibraryChatCompletion');
+const { Model } = require('./Model');
+
+class AiBrary {
+  constructor({ apiKey, ...kwargs }) {
+    if (!apiKey) {
+      apiKey = process.env.AIBRARY_API_KEY;
+    }
+    if (!apiKey) {
+      throw new Error(
+        'The api_key client option must be set either by passing apiKey to the client or by setting the AIBRARY_API_KEY environment variable'
+      );
+    }
+
+    this.apiKey = apiKey;
+    this.baseUrl = 'https://api.aibrary.dev/v0';
+
+    openai.apiKey = this.apiKey;
+    openai.baseUrl = this.baseUrl;
+
+    this.chat = {
+      completions: new AibraryChatCompletion(this)
+    };
+  }
+
+  async getAllModels({ returnAsObjects = true, filterCategory = null } = {}) {
```

Comment:
let's rename this to getModelsAsJson and create another method getModelsAsObject. then you do not need to pass returnAsObjects.
also let's rename the other prama to filters which can have a key for category. in future you might add more filters
tip: boolean params can be code smell.